nipy/labs/viz_tools/test/test_activation_maps.py

```diff
@@ -3,6 +3,7 @@
 # vi: set ft=python sts=4 ts=4 sw=4 et:
 import tempfile
 
+from mock import patch
```

Comment:
awesome!  even if installed by travis, I don't see it used anywhere else :-/

try following
```python
try:
    from mock import patch
except ImportError:
    patch = None

```
then use patch not as a decorator but as a context handler within the test and decorate test with
```python
@skipif(not patch)
```
so the whole thing doesn't fail if mock is N/A